src/main.rs

```diff
@@ -81,14 +81,16 @@ fn watch() -> notify::Result<()> {
     let mut watcher: RecommendedWatcher = Watcher::new(tx, Duration::from_secs(2))?;
     watcher.watch("./exercises", RecursiveMode::Recursive)?;
 
-    let _ignored = verify();
+    let _ignored = verify(None);
 
     loop {
-        match rx.recv() {
+        match rx.recv() {     
             Ok(event) => match event {
                 DebouncedEvent::Create(b) | DebouncedEvent::Chmod(b) | DebouncedEvent::Write(b) => {
                     if b.extension() == Some(OsStr::new("rs")) {
-                        let _ignored = verify();
+                        println!("----------**********----------");
+                        println!();
```

Comment:
Is it a new line? Why not use `\n` in the macros above?

Reply:
Fair enough, I was just following the precedent set here: https://github.com/rust-lang/rustlings/blob/6cd42bb8210b0e687942e388d66b07f286fe6bb6/src/main.rs#L39-L47